src/js/containers/SharedContainers/SubTierAgencyListContainer.jsx

```diff
@@ -34,13 +34,13 @@ class SubTierAgencyListContainer extends React.Component {
 	dataFormatter(item, input) {
 		return {
 			label: item.agency_name,
-			value: item.agency_code
+			value: {value:item.agency_code, priority:item.priority}
```

Comment:
will changing the formatter here change anything in typeahead in the function at line 82?